docs/packaging-applications/build-servers/tfs-azure-devops/using-octopus-extension/build-versions-in-team-build.md

```diff
@@ -46,86 +46,25 @@ Other extensions such as [gitversion](https://github.com/GitTools/GitVersion) ca
 
 ## Packaging in Team Build
 
-There are two standard ways to package your build output in Team Build. The first is to use [OctoPack](/docs/packaging-applications/create-packages/octopack/index.md), and the second is to use the build tasks in the [Octopus marketplace extension](https://marketplace.visualstudio.com/items?itemName=octopusdeploy.octopus-deploy-build-release-tasks).
-
-You can mix and match these two methods. For example, you might prefer to use OctoPack and the `/p:RunOctoPack` MS Build argument, but then use the Push and Create Release tasks from the extension.
-
-:::warning
-**OctoPack and .NET Core**
-OctoPack is not compatible with .NET Core applications. If you want to package .NET Core applications see [create packages with the Octopus CLI](/docs/packaging-applications/create-packages/octopus-cli.md).
-:::
-
-## Versioning when using OctoPack
-
-Unless told otherwise, OctoPack will attempt to set the package version itself. You can see exactly how it tries to find a version [on this documentation page](/docs/packaging-applications/create-packages/octopack/index.md#UsingOctoPack-Versionnumbers).
-
-More often than not, the version number will come from the `[assembly: AssemblyVersion]` attribute found in the `AssemblyInfo.cs` file (usually in the `Properties` folder of your project).
-
-The assembly version number doesn't always behave the way we want. It may not change, and if it does, it may not change sequentially.
-
-Setting the `AssemblyVersion` number to `x.y.*` will [fix the two most common problems here](http://stackoverflow.com/questions/356543/can-i-automatically-increment-the-file-build-version-when-using-visual-studio). A version like `1.0.*` will result in an always-increasing version number.
-
-1. If the version number is fixed (e.g. `1.0.0.0`), Octopus will complain that the package number already exists.
-2. If the version number only changes "revision" number (e.g. `1.0.0.*`), numbers can repeat.
-
-One problem remains - particularly for web projects. If you only change content files, triggering a new build _won't update the version number_ because Team Build realizes no compilable code has changed. OctoPack won't get a new version number!
-
-### OctoPack recommendation
-
-To solve all these issues, we recommend using the `/p:OctoPackPackageVersion` argument when using OctoPack. This lets you specify the version directly, and you can use build variables to specify the version number.
+As mentioned above the recommended approach to package your application is to use the [Archive Files](http://go.microsoft.com/fwlink/?LinkId=809083) task.
```

Comment:
this is true for packaging applications. Maybe we should also mention `dotnet pack` if the target is a component/library?

Reply:
I don't think so, that IMO would be part of the build process, not the release pipeline, I understand you can pretty much do it all in the same pipeline, but still, there is a difference